sys/vane/jael.hoon

```diff
@@ -1127,34 +1404,56 @@
         [kyz ..file]
       ::
       ::  sanity check, should never fail if we operate correctly
+      ::  XX is this true in the presence of reorgs?
```

Comment:
Pretty sure that with @philipcmonk's recent changes we now account for reorgs? I'd expect the `?>` below to now go a rewind in case a reorg is detected though.

Reply:
I believe this check is valid again, since restoring a snapshot reverts `latest-block`, and we intercept `removed` cards earlier.  At any rate, if it isn't true, I would like to investigate that case.

I'll reinstate it for now.